docs/index.md

```diff
@@ -31,8 +31,7 @@ Showdown can be used on the client-side (in the browser) or server-side (with No
 
 ## Installation
 
-To install Showdown, follow the instructions from the [Installation guide](installation.md).
-
+To install Showdown, follow the instructions from the [Installation guide](https://showdownjs.com/docs/quickstart/#installation).
```

Comment:
An alternative link could be https://github.com/showdownjs/showdown#installation but I think it's better to stay on the showdownjs website.

Reply:
@cadamini Slightly improved your PR. Please check #944